TheWarWithin/Priorities/MageFrost.simc

```diff
@@ -1,24 +1,27 @@
+actions.precombat+=/variable,name=time_to_coc,condition=talent.frostfire_bolt.enabled,value=10,value_else=23
 actions.precombat+=/arcane_intellect
 actions.precombat+=/mirror_image
-actions.precombat+=/blizzard,if=active_enemies>=2&talent.ice_caller&!talent.fractured_frost|active_enemies>=3
-actions.precombat+=/frostbolt,if=active_enemies<=2
+actions.precombat+=/blizzard,if=active_enemies>=2&talent.ice_caller&!talent.fractured_frost|active_enemies>=4
+actions.precombat+=/frostbolt,if=active_enemies<=3
 
 actions+=/counterspell
 actions+=/call_action_list,name=cds
-actions+=/run_action_list,name=aoe,strict=1,if=active_enemies>=7|active_enemies>=3&talent.ice_caller
-actions+=/run_action_list,name=ss_cleave,strict=1,if=active_enemies>=2&active_enemies<=3&talent.splinterstorm
-actions+=/run_action_list,name=cleave,strict=1,if=active_enemies>=2&active_enemies<=3
-actions+=/run_action_list,name=ss_st,strict=1,if=talent.splinterstorm
-actions+=/run_action_list,name=st
+actions+=/call_action_list,name=aoe,strict=1,if=active_enemies>=7|active_enemies>=3&!talent.frostfire_bolt|active_enemies>=4
+actions+=/call_action_list,name=cleave_ff,strict=1,if=active_enemies>=2&active_enemies<=3&talent.frostfire_bolt
+actions+=/call_action_list,name=cleave_ss,strict=1,if=active_enemies>=2
+actions+=/call_action_list,name=st_aoebuild,strict=1,if=talent.splinterstorm&!(talent.cold_front&talent.slick_ice&talent.deaths_chill&talent.frozen_touch)|talent.frostfire_bolt&!talent.deep_shatter
+actions+=/call_action_list,name=st_ff,strict=1,if=talent.frostfire_bolt
+actions+=/call_action_list,name=st_ss
```

Comment:
We could save some CPU cycles by making this one `strict=1,if=!talent.frostfire_bolt` unless you want `st_ss` list items tested after `st_ff`.

Reply:
st_ff probably shouldn't ever completely fail, but will add for consistency.